styles/typography/_hyperlinks.scss

```diff
@@ -0,0 +1,15 @@
+/**
+ * Hyperlinks
```

Comment:
it's an anchor?

Reply:
Semantics -- I removed the comments anyway based on Brady's comment.

Reply:
I was more on about the file name